Makefile.coq.local

```diff
@@ -0,0 +1,19 @@
+GLOBFILES = $(VFILES:.v=.glob)
+CSSFILES = extra/coqdoc.css extra/coqdocjs.css
+JSFILES = extra/config.js extra/coqdocjs.js
+HTMLFILES = extra/header.html extra/footer.html
+COQDOCDIR = website
```

Comment:
Since the website target does nothing other than building the usual coqdocjs HTML documentation, maybe it would be a good idea to use the standard target name used for this purpose and adapt the meta data (documentation, `README.md`) accordingly.

Reply:
Unfortunately, there is no way to "hook into" the `coq_makefile` target `html` via `Makefile.coq.local` and add additional actions, such as copying the JavaScript and CSS files. In fact, the `html` target just looks like this:
```make
html: $(GLOBFILES) $(VFILES)
        $(SHOW)'COQDOC -d html $(GAL)'
        $(HIDE)mkdir -p html
        $(HIDE)$(COQDOC) \
                -toc $(COQDOCFLAGS) -html $(GAL) $(COQDOCLIBS) -d html $(VFILES)
```
Moreover, there are actually several additional ways to generate Coq HTML documentation nowadays, such as Xavier Leroy's [coq2html](https://github.com/xavierleroy/coq2html) and [Proviola](https://github.com/coq-community/manifesto/issues/16).

Hence, the directory structure I use in some other coq-community projects I maintain is like this:
```
docs/coqdoc
docs/proviola
docs/coq2html
```
... and the corresponding actions are `make coqdoc`, `make proviola` and so on. The HoTT project is even more ambitious and adds a suffix `-html`, since you can do PDFs via coqdoc.

Does this sound reasonable?

Reply:
I am happy with `make coqdoc`. I don't plan on adding anything other than CoqDocJS, so take whatever output location makes things more homogeneous within coq-communit to ease maintenance and deployment of the html documentation.